live/app/routes/courses/create-assessment.js

```diff
@@ -2,24 +2,15 @@ import BaseRoute from 'pix-live/routes/base-route';
 
 export default BaseRoute.extend({
 
-  model(params) {
-    return {
-      courseId: params.course_id
-    };
-  },
-
-  afterModel(model) {
-
+  redirect(course) {
```

Comment:
Pourquoi créer une méthode redirect dans cette route ?

Reply:
C'est dans le cas où on doit rediriger vers une route qui requiert les mêmes modèles que la route initiale, ça permet d'éviter de recharger "again and again" les modèles (qu'on a déjà). cf. https://guides.emberjs.com/v2.16.0/routing/redirection/#toc_child-routes